frontend/src/metabase/static-viz/lib/axes.js

```diff
@@ -14,10 +14,6 @@ export const getXTickWidthFromValues = (values, maxWidth, fontSize) => {
   return Math.min(tickWidth, maxWidth);
 };
 
-export const getRotatedXTickHeight = tickWidth => {
```

Comment:
This can now be removed as we can just use the `xTickWidth` directly since we now rotate them 90 degrees instead of 45 degrees. So I inlined its value instead and remove this function.